src/main/java/programminglife/gui/controller/GraphController.java

```diff
@@ -56,13 +53,25 @@ public void draw(int centerNode, int radius) {
                 drawEdge(drawableNode, child);
             }
         }
+
+        DrawableNode drawableCenterNode = subGraph.getNodes().get(centerSegment);
+        drawableCenterNode.setFill(Color.ORANGE);
```

Comment:
A generic method to highlight a specific (Drawable)Node in a specified color would be useful. This could also be used to higlight groups (e.g. genomes).